src/main/java/org/webjars/taglib/LocateTag.java

```diff
@@ -39,6 +39,15 @@
 	private Iterator<String> result;
 
 	/**
+	 * Whether the path we're looking for do not specify the version number.
```

Comment:
Sorry for the grammar issues (my English is not very good) but the point here is the contrary.  The path you write in the tag (the path you are looking for) does not include version information.  The resulting path after the tag code runs is the fully qualified path to the resource (including version information).  So, I think, that the comment should be something like "Whether the path we're looking for does not contain the version number."